<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferFunctionEditor</class>
 <widget class="QWidget" name="TransferFunctionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Default Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="rgbaLayout"/>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="redLayout"/>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="greenLayout"/>
   </item>
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="blueLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
